docs/en/operation/Metrics-System.md

```diff
@@ -126,3 +126,85 @@ to the under store.
 
 Tags are additional pieces of metadata for the metric such as user name or under storage location.
 Tags can be used to further filter or aggregate on various characteristics.
+
+## Setting up Grafana for Alluxio
+
+Grafana is a metics analytics and visualization software used for visualizing time series
+data. You can use Grafana to better visualize the various metrics that Alluxio collects, and the software
+allows users to more easily see what's going on behind the scenes in Alluxio.
+
+Before setting up Grafana a monitoring tool must be set up that Grafana will pull metrics from. Alluxio
+supports exporting metrics to `Prometheus` and `Graphite`, so set up for both will be shown.
+
+### Setting up Prometheus
+
+Installing Prometheus using the instructions [here](https://prometheus.io/docs/prometheus/latest/installation/).
+
+After installing Prometheus, it must be configured to get metrics from Alluxio. First open the
+`prometheus.yml` file that Prometheus calls when starting. Next put the following sections under the
+`scrape_configs:` section of the Prometheus configuration.
+
+```
+scrape_configs:
+  - job_name: 'alluxio master'
+    metrics_path: '/metrics/prometheus'
+    static_configs:
+    - targets: ['<alluxio_master_hostname>:19999']
+
+  - job_name: 'alluxio worker'
+    metrics_path: '/metrics/prometheus'
+    static_configs:
+    - targets: ['<alluxio_worker_hostname>:30000']
+```
+
+Jobs are the various locations Prometheus scrapes for metrics. For Alluxio we tell Prometheus to get data
+from `localhost:19999/metrics/json` and `localhost:30000/metrics/json` the locations that Alluxio stores
+metrics in for the Master and Worker.
+
+### Setting up Graphite
+
+Install Graphite using the instructions [here](https://graphite.readthedocs.io/en/latest/install.html).
+
+Once Graphite is installed it now must be configured with Alluxio.
+
+### Configure Monitoring Tools with Alluxio
+
+Next the Alluxio configuration file must be set up. This sends the Alluxio metrics to the monitoring
+software. In `./${ALLUXIO_HOME}/conf/metrics.properties` open `metrics.properties` or create the file
+if it doesn't exist. Once open if using Prometheus type:
+
+```
+metrics.sink.PrometheusMetricsServlet.path=/metrics/prometheus
+```
+
+Or if using Graphite:
+
+```
+alluxio.sink.graphite.class=alluxio.metrics.sink.GraphiteSink
+*alluxio.sink.graphite.host=localhost
+alluxio.sink.graphite.port=2003
+alluxio.sink.graphite.period=10
+master.source.jvm.class=alluxio.metrics.source.JvmSource
```

Comment:
is this necessary only for graphite? or is it used in both? What exactly does this do?

Reply:
It's necessary for Graphite and I believe it points Graphite to the metric sources for the master and worker